UnhollowerBaseLib/Runtime/VersionSpecific/MethodInfo/Interfaces.cs

```diff
@@ -4,24 +4,13 @@ namespace UnhollowerBaseLib.Runtime.VersionSpecific.MethodInfo
 {
     public interface INativeMethodInfoStructHandler : INativeStructHandler
     {
-        INativeMethodInfoStruct CreateNewMethodStruct();
+        INativeMethodInfoStruct CreateNewStruct();
         unsafe INativeMethodInfoStruct Wrap(Il2CppMethodInfo* methodPointer);
-        IntPtr GetMethodFromReflection(IntPtr method);
```

Comment:
Yeah it can be reverted, I'll need to add support for generating these methods as the generator I made doesn't support inserting methods into the handler class.

Reply:
I should also probably dump a list of methods in the il2cpp api

Reply:
Seems like the specific version of Unity where I needed to manually resolve the MethodInfo is lost. It's probably better to just test and fix it later down the line when possible. As such, I'll mark this as done and merge the code in.